### PR TITLE
Add ScrollView.padding

### PR DESCRIPTION
**Pull request by abarth**
Uses SliverPadding to implementing paddding in ScrollView. Also, deploy
ScrollView in more places now that it implements padding.

Finally, remove loader_app.dart because it is not referenced.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/test/widgets/date_picker_test.dart [hunk 1]
```diff
@@ -34,5 +34,5 @@ void main() {
     expect(currentValue, equals(new DateTime(2014, 6, 9)));
     await tester.tap(find.text('30'));
     expect(currentValue, equals(new DateTime(2013, 1, 30)));
-  });
+  }, skip: true);
```
Comment (Hixie) [inline packages/flutter/test/widgets/date_picker_test.dart]: File a bug about whatever is blocking this one?

**Comment by Hixie** [comment]
LGTM, but travis is angry.

File: packages/flutter/test/widgets/date_picker_test.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/test/widgets/date_picker_test.dart]: The test was previously disabled.  It's busticated for a bunch of reasons.  I tried fixing them, but it got too big.

Comment (Hixie) [inline packages/flutter/test/widgets/date_picker_test.dart]: Ah, I see. LGTM.

**Pull request merged**